_targets.R

```diff
@@ -14,52 +15,129 @@ source('7_drivers_munge/src/GCM_driver_utils.R')
 
 targets_list <- list(
   # The input from our `scipiper` pipeline
+  tar_target(lake_centroids_sf_rds, gd_get('2_crosswalk_munge/out/centroid_lakes_sf.rds.ind'), format='file'),
+
+  # FOR NOW, FOR TESTING - get subset of lake centroids
+  tar_target(subset_lake_centroids_sf, split_lake_centroids(lake_centroids_sf_rds)),
+
+  # TODO: bring in the python workflow to create this shapefile of the reconstructed grid cells
+  # Load in the reconstructed grid shapefile as a file target
+  tar_target(grid_cells_shp, '7_drivers_munge/in/gcm_grid_cells.shp', format='file'),
+
+  # load the grid cell shapefile using readOGR
+  # (so that we get the custom LCC projection information)
+  tar_target(grid_cells_ogr, rgdal::readOGR(grid_cells_shp)),
```

Comment:
I believe she did have to specifically switch because there were some issues with using `st_read()` with this shapefile. The shapefile was created using some Python code for now because Hayley was more comfortable making the grid that way. We were saving as a shapefile and importing, but planning to integrate that actual code with this repo as a later step. I wasn't aware of the technical concerns with `rgdal` but maybe we should plan to address the `st_read()` issues rather than use the workaround. 

@hcorson-dosch could you share the error you were getting?

Reply:
Ok, I was wondering about the Python addition here too. Is that a temporary workaround (building the grid w/ python) or something that is planned to include in the pipeline? I'm wondering about mixing languages in this case since the grid (hopefully?) can be created in `sf`.

Reply:
Hayley wrote in Python originally because it was quicker for her to implement. I think we could probably rebuild with `sf` instead, but wasn't sure if we should prioritize doing that or just try to add in a target that kicks off Python code.

Reply:
Sorry - didn't have my email open, so didn't see these notifications until now. I couldn't read in the file w/ `sf` alone b/c it didn't read in the projection information. I'm happy to re-write my code in R.

Reply:
Not sure what happened - I was pretty sure I checked with `sf::st_crs()` and it hadn't pulled the right crs info, but it's working now, so I switched to only using `sf`

Reply:
oh! Maybe it was updating `sf` that fixed it!

Reply:
Oh I bet that is what fixed it!

Reply:
Also, just to clarify -- I used Python yes, because I am more comfortable working with _spatial_ information in Python than in R, but also because I have more confidence in Python than R for spatial work, and I feel that the approach that I took in Python is more robust than the `st_make_grid` approach, which I did review before starting this work. I will however, re-write it in R, and see if I can adapt the `st_make_grid` approach. I think that makes sense in order to avoid having to incorporate python into this repo.

Reply:
Thanks for clarifying. Can you commit the Python code to the repo even if you use a separate commit to delete/replace it later? Then others in the project can learn more about the differences. If the code stays local, we have less of an opportunity to understand pros/cons to the R/py solution to various spatial problems. And if you may have a tendency to tidy-up before committing in this case, I'd suggest you skip that in favor of more iteration and trust that we've got context to be able to interpret WIP efforts.

Reply:
Yes for sure - I'd intended to do so as a separate PR, so that Lindsay could keep moving forward with munging, but can add that now.

Reply:
Right now it is in a notebook, so includes some code that wouldn't be included in the final set of functions, if this were to be incorporated into the pipeline. I've left the cell results in the notebook, so hopefully you can see the output.

Reply:
Notebook is [here](https://github.com/hcorson-dosch/lake-temperature-model-prep/blob/gcm_driver_data_munge_pipeline/7_drivers_munge/in/Reconstruct_GCM_grid.ipynb)